app/views/tag/blog.html.erb

```diff
@@ -7,9 +7,9 @@
     <% if params[:id] == "blog" %>
       <br />
       <p><%= t('tag.blog.stories_from_community') %></p>
-      <p><a href="/wiki/blog"><%= t('tag.blog.about_the_blog') %></a></p>
-      <p><a href="/research"><%= t('tag.research') %></a></p>
-      <p><a href="/tools"><%= t('tag.methods') %></a></p>
+      <p><a href="/wiki/blog"><%= t('tag.blog.about_the_blog') %>  </a> |
```

Comment:
Here, is the extra space before the `</a>` necessary? I don't think it'll cause trouble, but just wondering.

Also, I think it might read more cleanly with the `<p>` and `</p>` on their own lines, with the `<a>` tags indented one more level, know what I mean? 

Of course, this is all style preference, and while it's important to keep our codebase clean and readable, your PR should work fine. Would you like to make a followup commit before I merge?